db/users.js

```diff
@@ -1,9 +1,9 @@
 const client = require('./client')
 const bcrypt = require('bcrypt');
+const SALT_COUNT = 10;
```

Comment:
Nice job defining `SALT_COUNT` to the top

Reply:
Gold star for us!!